Source/CarthageKit/Git.swift

```diff
@@ -256,20 +255,28 @@ public func contentsOfFileInRepository(_ repositoryFileURL: URL, _ path: String,
 /// will be created.
 ///
 /// Submodules of the working tree must be handled seperately.
-public func checkoutRepositoryToDirectory(_ repositoryFileURL: URL, _ workingDirectoryURL: URL, revision: String = "HEAD") -> SignalProducer<(), CarthageError> {
+public func checkoutRepositoryToDirectory(_ repositoryFileURL: URL, _ workingDirectoryURL: URL,
+                                          revision: String = "HEAD") -> SignalProducer<(), CarthageError> {
 	return SignalProducer.attempt { () -> Result<[String: String], CarthageError> in
 			do {
 				try FileManager.default.createDirectory(at: workingDirectoryURL, withIntermediateDirectories: true)
 			} catch let error as NSError {
-				return .failure(CarthageError.repositoryCheckoutFailed(workingDirectoryURL: workingDirectoryURL, reason: "Could not create working directory", underlyingError: error))
+				return .failure(
+					CarthageError.repositoryCheckoutFailed(
+						workingDirectoryURL: workingDirectoryURL,
+						reason: "Could not create working directory", underlyingError: error
+					)
+				)
 			}
 
 			var environment = ProcessInfo.processInfo.environment
 			environment["GIT_WORK_TREE"] = workingDirectoryURL.path
 			return .success(environment)
-		}
-		.flatMap(.concat) { environment in launchGitTask([ "checkout", "--quiet", "--force", revision ], repositoryFileURL: repositoryFileURL, environment: environment) }
-		.then(SignalProducer<(), CarthageError>.empty)
+	}
+	.flatMap(.concat) { environment in
+		launchGitTask([ "checkout", "--quiet", "--force", revision ], repositoryFileURL: repositoryFileURL, environment: environment)
+	}
+	.then(SignalProducer<(), CarthageError>.empty)
```

Comment:
👍 for attempt in a new line. Otherwise I'm running into `closure_end_indentation` warnings from SwiftLint. I could turn them off, of course, but I like the latter solution more. Done that.